native/src/main/java/io/ballerina/stdlib/http/transport/message/HttpCarbonMessage.java

```diff
@@ -144,6 +145,9 @@ public synchronized void addHttpContent(HttpContent httpContent) {
     public HttpContent getHttpContent() {
         HttpContent httpContent = this.blockingEntityCollector.getHttpContent();
         this.contentObservable.notifyGetListener(httpContent);
+        if (httpContent != null) {
```

Comment:
`blockingEntityCollector` can contain multiple http contents right? In this case we only consider 1 content and set it to the carbon msg. Is that correct?